kotest-assertions/kotest-assertions-shared/src/commonMain/kotlin/io/kotest/assertions/errorScope.kt

```diff
@@ -0,0 +1,33 @@
+package io.kotest.assertions
+
+import io.kotest.common.ExperimentalKotest
+import io.kotest.fp.Try
+
+internal typealias Failures = List<Throwable>
+internal typealias Assertions = Int
+
+@ExperimentalKotest
+suspend fun <T> errorScope(block: suspend () -> T): Triple<Try<T>, Failures, Assertions> {
```

Comment:
This is not clear if you were to read it new. I think its worth really commenting on what's going on.

Reply:
Is error scope the right word? Like assertionsScope ?
It's counting assertions as well, not just errors.